docs/syntax-reference.md

```diff
@@ -337,6 +226,121 @@ The WITH clause supports the following properties:
 | TIMESTAMP               | Associates the message timestamp in the Kafka topic with a column in the KSQL table. Time-based operations such as windowing will process a record according to this timestamp. |
 
 
+### DESCRIBE
+
+**Synopsis**
+
+```sql
+DESCRIBE [EXTENDED] (stream_name|table_name);
+```
+
+
+**Description**
+
+* DESCRIBE: List the columns in a stream or table along with their data type and other attributes.
+* DESCRIBE EXTENDED: Display DESCRIBE information with additional runtime statistics, Kafka topic details, and the
```

Comment:
An intentional side effect of this bullet list is that "DESCRIBE EXTENDED" (unlike "DESCRIBE [EXTENDED]" in the synopsis) can be found via `Ctrl-F`-ing the document. ;-)